### ISSUE TITLE
perf: Fine-tune top convolutional blocks with low learning rate schedule

### ISSUE THREAD

**Issue by Bosaj** [Bosaj/Transfer-Learning-VGG16-Dog-Cat-Classification#3]
Unfreeze block5 conv layers with Adam optimizer (lr=1e-5) to maximize fine-grained validation accuracy.
